modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientHandlerModule.java

```diff
@@ -351,6 +352,7 @@ protected void initChannel(Channel ch) {
                             handler = messageHandler;
 
                             ch.pipeline().addLast(
+                                    new FlushConsolidationHandler(FlushConsolidationHandler.DEFAULT_EXPLICIT_FLUSH_AFTER_FLUSHES, true),
```

Comment:
Should we enable this on the client side too, in `TcpClientChannel`?

Reply:
Ignore my comments - it is already enabled in `NettyClientConnectionMultiplexer`.